Not sure why the Meson Python module can't find the installed pulsectl...

diff --git a/meson.build b/meson.build
index 1f84fe8..05567a4 100644
--- a/meson.build
+++ b/meson.build
@@ -22,7 +22,7 @@ endif
 
 pymod = import('python')
 python = pymod.find_installation('python3',
-  modules: ['click', 'ewmh', 'setproctitle', 'pulsectl']
+  modules: ['click', 'ewmh', 'setproctitle']
 )
 python.dependency()
 
